packages/patterns-react/src/patterns/SimpleLongForm/SimpleLongForm.js

```diff
@@ -13,7 +13,10 @@ import { settings } from 'carbon-components';
 import { featureFlag } from '@carbon/ibmdotcom-utilities';
 import { SIMPLELONGFORM } from '../../internal/FeatureFlags';
 import JumpLink from './JumpLink';
-import CardLink from './CardLink';
+// import CardLink from './CardLink';
+import { CardLink } from '@carbon/ibmdotcom-react';
+import '@carbon/ibmdotcom-styles/scss/components/card-link/index.scss';
```

Comment:
You want to import this in your `SCSS`.  Then you can use the mixin to include it with your theme. Otherwise, the dark theme won't work.

```scss
@import '@carbon/ibmdotcom-styles/scss/components/card-link/index';

.#{$prefix}--simplelongform--g100 {
  @include carbon--theme($carbon--theme--g100) {
    @include simplelongform;
    @include card-link;
  }
}
```

Reply:
Updated my comment above. `@include tile;` is required for the moment. I imagine we'll want to fix this moving forward.

Reply:
Importing the as mention above still ended up with error

```javascript
@import '../../globals/imports';
@import './simplelongform';
@import './jumplink';
@import './simplelink';
@import '@carbon/ibmdotcom-styles/scss/components/card-link/index';

.#{$prefix}--simplelongform--g100 {
  @include carbon--theme($carbon--theme--g100) {
    @include simplelongform;
    @include jumplink;
    @include simplelink;
    @include tile;
    @include card-link;
  }
}
```

<img width="710" alt="Screen Shot 2019-10-29 at 6 19 11 PM" src="https://user-images.githubusercontent.com/8072521/67813923-262d4a80-fa79-11e9-909b-605b49ed0c49.png">

Reply:
How could we fix this ?

Reply:
Is there any reason setting CardLink component background color as `background-color: $ui-01;` won't work ?

Reply:
The theme sets more than just the `background-color`. It also might not be right if you themed it in a different theme.  If Carbon changes the background then it won't change with it. `$ui-01` a different meaning/intent behind it.

I'm not sure why it's not working off of what I'm seeing here. I can try to take a look at it tonight though.

Reply:
Imported global CardLink component, we still need to figure out to make it work with different themes.
[commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/623e442313c5b2667c6db8fdd7bc7d5760e44b8f)

Reply:
I tried importing it from pattern's index.sass file
It works now. Thank you.
here is [the commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/d929b848869c81442388b722584681a6ffe38f70)

Reply:
@sannminwin Have you updated `@carbon/ibmdotcom-react` and `@carbon/ibmdotcom-styles` to the latest version?